### PR TITLE
How to add new benchmark operator workload to benchmark runner

### PR DESCRIPTION
**Pull request by ebattat**
This PR contain how to add new workload readme file

### CODE REVIEW

File: HOW_TO.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+7. Add workload folder path in [MANIFEST.in](MANIFEST.in), for each flavor 'func_ci', 'perf_ci', 'test_ci' add 2 paths: the workload path to 'workload_data_template.yaml' and path to 'internal_data' Pod and VM template yaml files
+8. For test and debug workload, need to configure [environment_variables.py](benchmark_runner/main/environment_variables.py)
+   1. Fill parameters: workload, kubeadmin_password, pin_node_benchmark_operator, pin_node1, pin_node2, elasticsearch, elasticsearch_port 
+   2. Run [main.py](/benchmark_runner/main/main.py)  and verify that the workload run correctly
+   3. The workload can be monitored and checked through 'current run' folder inside the run workload flavor (default flavor: 'test_ci')
+9. Open Kibana url and verify workload index populate with data:
+   1. Create the workload index: Kibana -> Hamburger tab -> Stack Management -> Index patterns -> Create index pattern -> workload-results -> timestamp -> Done
+   2. Verify workload-results index is populated: Kibana -> Hamburger tab -> Discover -> workload-results (index) -> verify that there is a new data
+
+## Add workload to grafana dashboard
```
Comment (RobertKrawitz) [inline HOW_TO.md:32]: Which Grafana instance?

File: HOW_TO.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. Create Elasticsearch data source
+   1. Grafana -> Configuration(setting icon) -> Data source -> add data source -> Elasticsearch
+      1. Name: Elasticsearch-workload-results
+      2. URL: http://elasticsearch.com:port
```
Comment (RobertKrawitz) [inline HOW_TO.md:36]: What exact URL?

File: HOW_TO.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+   1. Open grafana
+   2. Create(+) -> import -> paste [benchmark-runner-report.json](grafana/benchmark-runner-report.json) -> Load
+   3. Create panel from scratch or duplicate existing on (stressng/uperf)
+   4. Configure the workload related metrics
+   5. Save dashboard -> share -> Export -> view json -> Copy to clipboard -> override existing one [benchmark-runner-report.json](grafana/benchmark-runner-report.json)
+   
+## Monitor and debug workload
+1. git clone https://github.com/redhat-performance/benchmark-runner
+2. cd benchmark-runner
+3. There are 2 option to run workload:
```
Comment (RobertKrawitz) [inline HOW_TO.md:51]: "option" => "options"

File: HOW_TO.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+         4. `pin_node_benchmark_operator` - benchmark-operator node selector
+         5. `pin_node1` - workload first node selector
+         6. `pin_node2` - workload second node selector (for workload with client server i.e. uperf)
+         7. `elasticsearch` - elasticsearch url without http prefix
+         8. `elasticsearch_port`
+      2. Run [main.py](/benchmark_runner/main/main.py) 
+      3. Verify that benchmark-runner run the workload
+   2. Run workload through integration/unittest tests [using pytest]
+      1. Need to configure all mandatory parameters [test_environment_variables.py](tests/integration/benchmark_runner/test_environment_variables.py)
+         1. `runner_path` = path to local cloned benchmark-operator (i.e /home/user/)
```
Comment (RobertKrawitz) [inline HOW_TO.md:66]: "i. e." => "e. g."

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
Pretty minor stuff.

File: HOW_TO.md (hunk 1 above)
Comment (ebattat) [reply HOW_TO.md:32]: Func or Perf environment Grafana instance

File: HOW_TO.md (hunk 2 above)
Comment (ebattat) [reply HOW_TO.md:36]: The URLs are confidential, cannot expose it in public

File: HOW_TO.md (hunk 3 above)
Comment (ebattat) [reply HOW_TO.md:51]: Fixed

File: HOW_TO.md (hunk 4 above)
Comment (ebattat) [reply HOW_TO.md:66]: Fixed

File: HOW_TO.md (hunk 1 above)
Comment (RobertKrawitz) [reply HOW_TO.md:32]: At least specify that latter, then (and as much of a pointer as you can to where to find those).

File: HOW_TO.md (hunk 2 above)
Comment (RobertKrawitz) [reply HOW_TO.md:36]: Understood; see above.

**Comment by ebattat** [comment]
@RobertKrawitz, lets discuss it in the next meeting, I think that the best practice is to enrich the document while adding a new workload and see if there is a missing stuff or need to correct something. I have added the highlights and verified that add all the important things is there.

**Comment by RobertKrawitz** [comment]
> @RobertKrawitz, lets discuss it in the next meeting, I think that the best practice is to enrich the document while adding a new workload and see if there is a missing stuff or need to correct something. I have added the highlights and verified that add all the important things is there.

I concur.  So let's keep this one on hold for the time being (and please fix those two other nits and say what you can to help guide someone to the correct grafana instance).

File: HOW_TO.md (hunk 4 above)
Comment (RobertKrawitz) [reply HOW_TO.md:66]: Please push the fix.

**Review by RobertKrawitz — APPROVED** [review]
Looks good.

**Pull request merged**